src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -493,7 +493,16 @@ private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception
 								}
 							}
 							catch (Exception e) {
-								log.error ("Error executing SQL on temporary SVN SQLite database '"+ sqliteConnectionUrl + "': "+ e);
+								StringBuilder errorSb=new StringBuilder(300);
+								errorSb.append("Error executing SQL on temporary SVN SQLite database '");
+								errorSb.append(sqliteConnectionUrl);
+								errorSb.append("'");
+								if (log.isDebugEnabled()) {
```

Comment:
So the reason I changed to only log the exception when debug is active is that a number of months ago when we started considering promoting a bunch of the scanners the comment was made that they shouldn't be causing exceptions. Granted this one is explainable but I think psiinon's point was twofold:

1. That exceptions may lead to user complaints/tickets. (As here.)
2. That exceptions may suggest something is wrong with the plugin. (Not true here.)

I'm happy with it being included in this case. Just want to be sure I'm adhering to the spirit of the original direction (which maybe I misinterpreted? @psiinon ?)